ERCS/erc-7941.md

```diff
@@ -0,0 +1,573 @@
+
+```solidity
+mapping(address => Object.Member) public memberInfos; // Stores Smart Common members and their dividend ratios
+uint32 public proposalLifetime; // Validity period of Smart Common proposals
+uint32 public proposalCoolingPeriod; // Cooling period for Smart Common proposals
+uint16 public strategy; // Pass rate for Smart Common proposals
+mapping(uint => File) public logoStorages; // Storage for Smart Common logos
+```
+- Verify if it is scc0 v1 version using the following methods:
+```solidity
+interface IDaism {
+    //check the address whether daism sc type of dapp
+    function dappToSC(address dApp) external view returns (uint);
+}
+
+// Check if a dApp is idaism dapp(scc0 v1 version)
+//daismAddress=0xdFBF69B7E5366FB3001C9a214dd85c5FE3f90bAe
+function isDaismSC(address _dApp) external view returns (bool) {
+     return IDaism(daismAddress).dappToSC(_dApp)>0;
+}
+```
+
+**Additional Governance and Operational Parameters of SCC0 V1**
+
+Beyond the core license and reference implementation contracts, SCC0 includes further parameters to support decentralized governance and community interaction:
+
+- Smart Common Structure:
+    A predefined structure (`SCInfo`) storing metadata such as the name, symbol, description, manager address, version, and type.
+    
+- Member and Proposal Management:
+    Mappings for recording member details (such as dividend ratios), proposal lifetimes, cooling periods, and decision-making strategies.
+    
+- Branding and Identity:
+    Storage mappings for smart common logos that enhance identity and trust within the ecosystem.
+
+**Reward Distribution Mechanism**
+
+To support SCC0-compliant projects, an upgradeable reward distribution system is introduced by SSC0 V1:
+
+1. Maintain an array to store external accounts eligible for rewards and their allocation percentages.
+2. Rewards are not directly sent to external accounts. Instead, they are deposited into the public governance contract.
+3. External accounts can withdraw funds any time.
+
+```solidity
+mapping(address => Object.Member) public memberInfos; // Stores smart common members and their dividend ratios
+uint32 public proposalLifetime; // Validity period of smart common proposals
+uint32 public proposalCoolingPeriod; // Cooling period for smart common proposals
+uint16 public strategy; // Pass rate for smart common proposals
+mapping(uint => File) public logoStorages; // Storage for smart common logos 
+```
+
+The reason why neither SSC0 V1 nor SSC0 V2 has introduced "detailed reward rules from Satoshi UTO Fund for smart commons" is that we can neither implement such measures through any centralized review panel approach, nor determine reward amounts through community voting using wallet addresses. The latter approach is even worse - it constitutes a pseudo-decentralized method that would only be employed by self-deceivers or even scammers. We expect some dAIpp take this work in the future, from valuation to prize management.
```

Comment:
> The reason why [...]

Content explaining choices made within the document itself belongs in the Rationale section.